cli/src/progress.rs

```diff
@@ -45,7 +31,7 @@ impl Progress {
     pub fn new<ProgressFnT, StatisticsT>(
         progress_fn: ProgressFnT,
         statistics: &Arc<StatisticsT>,
-        target_value: u64,
```

Comment:
Passing in `None` means an unbounded progress bar (e.g. spinner) -- we don't have total counts for how many comments are in the range (as it's a per source operation)